Soccar [Game-Server]/src/nl/soccar/gameserver/controller/socnet/message/SpawnObstacleMessage.java

```diff
@@ -0,0 +1,61 @@
+package nl.soccar.gameserver.controller.socnet.message;
+
+import nl.soccar.gameserver.controller.socnet.message.handler.SpawnObstacleMessageHandler;
+import nl.soccar.library.enumeration.ObstacleType;
+import nl.soccar.socnet.message.Message;
+import nl.soccar.socnet.message.MessageConstants;
+import nl.soccar.socnet.message.MessageEvent;
+
+/**
+ * @author PTS34A
+ */
+@MessageEvent(id = MessageConstants.SPAWN_OBSTACLE_MESSAGE_ID, handler = SpawnObstacleMessageHandler.class)
+public final class SpawnObstacleMessage extends Message {
+
+    private final float x;
+    private final float y;
+    private final float angle;
+
+    private final float width;
+    private final float height;
+    private final ObstacleType type;
+
+    public SpawnObstacleMessage(float x, float y, float angle, float width, float height, ObstacleType type) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)